components/eam/src/physics/crm/crm_output_module.F90

```diff
@@ -90,6 +90,8 @@ module crm_output_module
       real(crm_rknd), allocatable :: q_vt_tend (:,:)       ! CRM output tendency for QT  variance transport
       real(crm_rknd), allocatable :: t_vt_ls   (:,:)       ! large-scale LSE variance transport tendency from GCM
       real(crm_rknd), allocatable :: q_vt_ls   (:,:)       ! large-scale QT  variance transport tendency from GCM
+      real(crm_rknd), allocatable :: u_vt_tend (:,:)       ! CRM output tendency for U variance transport
```

Comment:
the same cosmetic change here too. grouping *_vt_ls together and grouping *_vt_tend together.

Reply:
done.